oap-server/server-receiver-plugin/skywalking-trace-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/trace/provider/parser/listener/endpoint/MultiScopesSpanListener.java

```diff
@@ -179,13 +187,13 @@ public void parseEntry(SpanDecorator spanDecorator, SegmentCoreInfo segmentCoreI
             for (KeyStringValuePair tag : spanDecorator.getAllTags()) {
                 if (SpanTags.DB_STATEMENT.equals(tag.getKey())) {
                     String sqlStatement = tag.getValue();
-                    if (!StringUtil.isEmpty(sqlStatement) && sqlStatement.length() > config.getMaxSlowSQLLength()) {
-                        statement.setStatement(sqlStatement.substring(0,config.getMaxSlowSQLLength()));
-                    }
-                    else {
+                    if (StringUtil.isEmpty(sqlStatement)) {
+                        statement.setStatement("[No statement]/" + spanDecorator.getOperationName());
```

Comment:
You may not have the operation string, take a look a prev review comment.

Reply:
This commit committed before you comment.

Reply:
Ha, good to know.